# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/rep-gtk/no_double_definitions.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur rep-gtk-0.18/gtk-compat.c src.rep-gtk.1100023779.22603.1788690224/rep-gtk-0.18/gtk-compat.c
--- rep-gtk-0.18/gtk-compat.c	2002-07-12 06:19:22.000000000 +0200
+++ src.rep-gtk.1100023779.22603.1788690224/rep-gtk-0.18/gtk-compat.c	2004-11-09 19:26:34.955192600 +0100
@@ -63,22 +63,6 @@
 }
 
 GtkWidget*
-gtk_radio_menu_item_new_with_label_from_widget (GtkRadioMenuItem *group,
-						gchar            *label)
-{
-  GSList *g = group? gtk_radio_menu_item_group (group) : NULL;
-  return gtk_radio_menu_item_new_with_label (g, label);
-}
-
-GtkWidget*
-gtk_radio_menu_item_new_with_mnemonic_from_widget (GtkRadioMenuItem *group,
-						   gchar            *label)
-{
-  GSList *g = group? gtk_radio_menu_item_group (group) : NULL;
-  return gtk_radio_menu_item_new_with_mnemonic (g, label);
-}
-
-GtkWidget*
 gtk_radio_menu_item_new_from_widget (GtkRadioMenuItem *group)
 {
   GSList *g = group? gtk_radio_menu_item_group (group) : NULL;
diff -dur rep-gtk-0.18/rep-gtk.h src.rep-gtk.1100023779.22603.1788690224/rep-gtk-0.18/rep-gtk.h
--- rep-gtk-0.18/rep-gtk.h	2003-02-12 09:51:45.000000000 +0100
+++ src.rep-gtk.1100023779.22603.1788690224/rep-gtk-0.18/rep-gtk.h	2004-11-09 19:24:21.553472744 +0100
@@ -289,12 +289,6 @@
 			    guint32 activate_time,
 			    repv position);
 
-GtkWidget*
-gtk_radio_menu_item_new_with_label_from_widget (GtkRadioMenuItem *group,
-						gchar            *label);
-GtkWidget*
-gtk_radio_menu_item_new_with_mnemonic_from_widget (GtkRadioMenuItem *group,
-						   gchar            *label);
 GtkWidget* gtk_radio_menu_item_new_from_widget (GtkRadioMenuItem *group);
 GtkWidget* gtk_pixmap_new_interp (char *file, GtkWidget *intended_parent);
 
